packages/jest-jasmine1/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "jest-jasmine1",
+  "version": "0.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "src/index.js",
+  "dependencies": {},
+  "scripts": {
+    "test": "jest"
```

Comment:
this one also won't work because jest `0.9.2` is not published yet.
it needs `../../bin/jest.js` or npm link to run it, otherwise will fail on `jest.disableAutomock is undefined`